modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js

```diff
@@ -105,9 +105,11 @@ const TranslationAdminContent = ({
 							onActiveChange={setCreationMenuActive}
 							trigger={
 								<ClayButtonWithIcon
+									className="lfr-portal-tooltip"
 									disabled={availableLocales.length === 0}
 									small
 									symbol="plus"
+									title={Liferay.Language.get('add')}
```

Comment:
Why "add" and not "add language"? Is this key defined somewhere on the TM design or in any desgin pattern?

Reply:
This is the tooltip used in a lot of management toolbar button across the portal, but it seems pretty inconsistent, let's ask david, yeah